package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "sicksync",
-  "version": "1.1.16",
+  "version": "1.1.17",
```

Comment:
Not sure about this, but needs to be @1.1.18 after this is merged, so this will likely be blown away when tagged and published.